frontend/src/functions/Authentications.js

```diff
@@ -2,14 +2,17 @@ import { ref } from 'vue';
 import apiClient from '@/api';
 import { googleTokenLogin } from 'vue3-google-login';
 import { createPostRequest } from './HttpRequest';
-import { getCookie, setCookie, deleteCookie } from './cookies';
+import { getCookie, setCookie, deleteCookie } from './Cookies.js';
```

Comment:
```suggestion
import { getCookie, setCookie, deleteCookie } from './Cookies.js';
```
I'm not sure what happen here but './Cookies.js'; with capital C cause a Module not found error in my machine.

Reply:
I think it is something weird about git, I already changed its name to Cookie to follow other files. but Seems like it doesn't updates on git